src/panasonicmn_int.cpp

```diff
@@ -743,6 +808,8 @@ namespace Exiv2 {
         {0x0116, "RowsPerStrip", N_("Rows Per Strip"), N_("The number of rows per strip"), panaRawId, panaRaw, unsignedShort, -1, printValue},
         {0x0117, "StripByteCounts", N_("Strip Byte Counts"), N_("Strip byte counts"), panaRawId, panaRaw, unsignedLong, -1, printValue},
         {0x0118, "RawDataOffset", N_("Raw Data Offset"), N_("Raw data offset"), panaRawId, panaRaw, unsignedLong, -1, printValue},
+        {0x0119, "DistortionInfo", N_("Distortion Info"), N_("Distortion info"), panaRawId, panaRaw, undefined, -1, printDistortionInfo},
+        {0x011b, "ChromaticAberration", N_("Chromatic Aberration"), N_("Chromatic aberration"), panaRawId, panaRaw, undefined, -1, printChromaticAberration},
```

Comment:
For starters, I'd just add these w/ special printers. This at least provides access to the data, and lets 3rd party consumers interpret it (they'd just copy it in to a binary buffer representing a structure anyway).